tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/Main.java

```diff
@@ -0,0 +1,161 @@
+package fi.helsinki.cs.tmc.langs.util;
+
+import com.google.common.base.Optional;
+import fi.helsinki.cs.tmc.langs.ExerciseDesc;
+import fi.helsinki.cs.tmc.langs.RunResult;
+import fi.helsinki.cs.tmc.stylerunner.validation.ValidationResult;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+    final static String EXERCISE_PATH = "exercisePath";
+    final static String OUTPUT_PATH = "outputPath";
+    private static TaskExecutor executor = new TaskExecutorImpl();
+
+    public Main(TaskExecutor executor) {
+        Main.executor = executor;
+    }
+
+    public static void main(String[] args) {
+        if (args == null || args.length == 0) {
+            printHelp();
+        }
+        run(args);
+        System.exit(0);
+    }
+
+    private static void printHelp() {
+        System.out.println("\nUsage: Main <exercise path> <output path>\n"
+            + "\nOptions:\n"
+            + " --checkstyle <exercise path> <output path>\t\tRun checkstyle or similar plugin to project if applicable.\n"
+            + " --help\t\t\t\t\t\t\tDisplay help information.\n"
+            + " --preparesolution <exercise path>\t\t\tPrepare a presentable solution from the original.\n"
+            + " --preparestub <exercise path>\t\t\t\tPrepare a stub exercise from the original.\n"
+            + " --runtests <exercise path> <output path>\t\tRun the tests for the exercise.\n"
+            + " --scanexercise <exercise path> <output path>\t\tProduce an exercise description of an exercise directory.\n");
```

Comment:
I'd rather align these with spaces. Much easier to see that they're aligned correctly.